### PR TITLE
chore: prep v1.30

### PR DESCRIPTION
**Pull request by yquansah**
prep v1.30

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,24 @@
...
+- `git`: support SSH authentication (#2272)
+
+### Changed
+
+- `ui`: redux all flag and variant state management (#2301)
+
+### Fixed
+
+- readOnly config option should be read_only (#2298)
+- Audit log path fixes (#2295)
```
Comment (markphelps) [inline CHANGELOG.md:22]:
```suggestion
- Audit log path will be created if doesn't exist (#2295)
```

this was my bad on naming the PR

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,24 @@
 This format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+
```
Comment (markphelps) [inline CHANGELOG.md:6]:
```suggestion
```

we need to update the generator to remove this extra blank line, don't want to upset the linters

**Review by markphelps — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,24 @@
...
+### Added
+
+- add flag key to variant and boolean evaluation responses (#2318)
+- add subject to authentication metadata for auditing (#2299)
+- `git`: support SSH authentication (#2272)
+
+### Changed
+
+- `ui`: redux all flag and variant state management (#2301)
```
Comment (markphelps) [inline CHANGELOG.md:17]:
```suggestion
- `ui`: redux all flag and variant state management (#2301)
- dependency updates
```

lets also add dependency updates here

**Review by markphelps — COMMENTED** [review]
looks good thank. lets keep this branch open for now while we do the release

**Review by GeorgeMac — APPROVED** [review]
LGTM :shipit:

**Pull request merged**